### PR TITLE
Ei nextjs pages

### PR DESCRIPTION
**Pull request by DeveloperIkhalea**
## Overview

**Issue Type**

- [ ] Bug
- [x] Feature
- [ ] Tech Debt
- [ ] Docs
- [ ] Other

**Description**
A clear and concise description of the issue.
Created a ui folder and putting the code for the sidebar components, and also restructured the landing page and its components.
**Ticket Item**

**Figma link - optional**
**Spec link - optional**

**Steps to Reproduce Bug / Validate Feature / Confirm Tech Debt Fix**

**Previous behavior**
A clear and concise description of what was originally happening.

**Expected behavior**

**Screenshots & Videos**
If applicable, add screenshots and videos to help explain your issue type.

**Additional context**

### CODE REVIEW

**Review by jesse-chou — APPROVED** [review]
Looks good to me. Thanks Emmanuel

File: app/dashboard/Test/page.tsx
Comment (derekcoughlan) [inline app/dashboard/Test/page.tsx:1]: @DeveloperIkhalea - What is this page?

File: app/dashboard/layout.tsx
Comment (derekcoughlan) [inline app/dashboard/layout.tsx:1]: @DeveloperIkhalea is {children} not needed in this page? Do the children within this layout render properly?

File: app/dashboard/profile/page.tsx
Comment (derekcoughlan) [inline app/dashboard/profile/page.tsx:1]: @DeveloperIkhalea What is the profile page?

**Review by derekcoughlan — COMMENTED** [review]
Thanks Emmanuel! Just a few questions for you.

**Review by jesse-chou — CHANGES_REQUESTED** [review]
Changes needed. Format riles

**Review by derekcoughlan — APPROVED** [review]
Thanks!

**Review by jesse-chou — APPROVED** [review]
Thanks for the changes Emmanuel.

**Pull request merged**